### PR TITLE
added files to be ignored by .gitignore

### PR DESCRIPTION
**Pull request by giovannivelludo**
I was having problems with loading project settings on my machine, adding these files to .gitignore (which were ignored in our group's projects) seems to have fixed the issue.

### CODE REVIEW

**Comment by CremaLuca** [comment]
Interesting, how does ignoring two modules files fix the problem? Is there a known reason or just straight magic?

Also i'm declining the other pull request to update gradle because it's also on this one.

File: .gitignore [hunk 1]
```diff
@@ -38,6 +38,8 @@ captures/
 .idea/tasks.xml
 .idea/gradle.xml
 .idea/assetWizardSettings.xml
+.idea/modules.xml
+.idea/navEditor.xml
```
Comment (CremaLuca) [inline .gitignore]: Seen it also in https://gist.github.com/iainconnor/8605514
Might be a good thing to do then

**Comment by giovannivelludo** [comment]
@CremaLuca All I know is that Marco used a gitignore suggested by GitHub when he created the repository, while the gitignore we had in our group project was generated by Android Studio. I guess the second way to generate it is better because the way Android Studio works changes with updates, and GitHub might not be always up to date with Android Studio changes.

**Comment by Cogno-IDU** [comment]
Yup, we can either let android studio create it's own, or use the one in the link @CremaLuca posted.
What would you prefer? I'd use the one of the link

**Comment by Cogno-IDU** [comment]
Also if this problem is here, then it's also in [killer-app](https://github.com/Cogno-IDU/killer-app) and [network-dictionary](https://github.com/Cogno-IDU/network-dictionary)

**Comment by giovannivelludo** [comment]
I think we should use the one generated by Android Studio since we don't know if the one linked above will always be up to date.  
Please don't merge this pull request until I remove the files I added to .gitignore, which I figured out is not done automatically by git.

**Comment by giovannivelludo** [comment]
Ok, now the pull request can be merged.

**Comment by giovannivelludo** [comment]
Merge it then

**Comment by Cogno-IDU** [comment]
Was waiting for @CremaLuca to see, seems there's no reason to wait so I'll merge myself

**Pull request merged**